### PR TITLE
Leader Board: Add the new style

### PR DESCRIPTION
**Pull request by batoorsohail**
### In this project, we added the following:

- [x] Background linear color
- [x] Overflow to scroll down
- [x] New style 
- [x] Mobile version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +88,6 @@ Example commands:
   cd my-folder
   git clone https://github.com/batoorsohail/leaderboard.git
 ```
--
 
 ### Install
```
Comment (rustam-tolipov) [inline README.md:90]: @batoorsohail You did great job!

- [ ] I like the styling. However, we have some parts in your README.md are missing. It should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them

They are placed in this section:


```
Install
Install this project with:   ----> install with ?

Usage
To run the project, execute the following command: ----> what command ?

Run tests
To run tests, run the following command: ----> what command ?

Deployment
You can deploy this project using: ----> using what ?
```

Kindly fill gaps in this sections. Also you are free to remove unnecessary parts from it such as how to deply.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @batoorsohail,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rustam-tolipov) [reply README.md:90]: Also, in Acknowledgment:

```
I would like to thank... who/what?
```

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @batoorsohail, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**